<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>809</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>ClassSystem 管理后台</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="m_tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="m_tabStudents">
       <attribute name="title">
        <string>学生</string>
       </attribute>
       <widget class="QGroupBox" name="m_groupBoxStudents">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>281</width>
          <height>324</height>
         </rect>
        </property>
        <property name="title">
         <string>学生</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="3">
          <widget class="QTableWidget" name="m_studentsTable">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>0</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>120</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>0</number>
           </attribute>
           <column>
            <property name="text">
             <string>学号</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>姓名</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="m_btnAddStudent">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="m_btnRemoveStudent">
           <property name="text">
            <string>移除</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="m_btnClearStudents">
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>导入</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="m_tabLessons">
       <attribute name="title">
        <string>课程</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="m_tabNoticesEvents">
       <attribute name="title">
        <string>公告&amp;&amp;事件</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="m_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="m_actChangePwd"/>
   <addaction name="separator"/>
   <addaction name="m_actSave"/>
   <addaction name="m_actDrop"/>
  </widget>
  <action name="m_actChangePwd">
   <property name="icon">
    <iconset resource="../../assets/ClassAdmin.qrc">
     <normaloff>:/img/password.png</normaloff>:/img/password.png</iconset>
   </property>
   <property name="text">
    <string>更改密码</string>
   </property>
  </action>
  <action name="m_actSave">
   <property name="icon">
    <iconset resource="../../assets/ClassAdmin.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存修改</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="m_actDrop">
   <property name="icon">
    <iconset resource="../../assets/ClassAdmin.qrc">
     <normaloff>:/img/drop.png</normaloff>:/img/drop.png</iconset>
   </property>
   <property name="text">
    <string>放弃修改</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../assets/ClassAdmin.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_btnAddStudent</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addStudent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>869</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btnClearStudents</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearStudents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>1037</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>importStudents()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>927</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btnRemoveStudent</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeStudent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>1111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_actChangePwd</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetPwd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addStudent()</slot>
  <slot>clearStudents()</slot>
  <slot>importStudents()</slot>
  <slot>removeStudent()</slot>
  <slot>resetPwd()</slot>
 </slots>
</ui>
